.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+    "tabWidth": 4
```

Comment:
This is the only Prettier rule I specified, since most of the codebase seems to use 4 space tabs. The other defaults seem sane and generally line up with our code.